### PR TITLE
Improving the app creation scripts:

### PR DESCRIPTION
**Pull request by jmprieur**
## Purpose
Improving the app creation scripts:
- AzureAD PowerShell modules are installed with the scope of the current user if they are not yet installed on the machine

Also:
- adding more references to the the README.md
- simplifying the AppCreationScripts.md

## Does this introduce a breaking change?
```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```
## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code
Go through the README.md

### CODE REVIEW

File: AppCreationScripts/Configure.ps1 [hunk 1]
```diff
@@ -195,14 +195,19 @@ Function ConfigureApplications
    $dictionary = @{ "ClientId" = $clientAadApplication.AppId };
    UpdateTextFile -configFilePath $configFile -dictionary $dictionary
    Write-Host ""
-   Write-Host "IMPORTANT: Think of completing the following manual step(s) in the Azure portal":
+   Write-Host "IMPORTANT: Please follow the instructions below to complete a few manual step(s) in the Azure portal":
```
Comment (kalyankrishna1) [inline AppCreationScripts/Configure.ps1:198]: >Please follow the instructions below to complete a few manual step(s) [](start = 26, length = 69)

Complete the manual step(s) provided below

File: README.md [hunk 2]
```diff
@@ -175,16 +191,20 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
...
+- [Customizing Token cache serialization](https://aka.ms/msal-net-token-cache-serialization)
+- [Scenarios](https://aka.ms/msal-net-scenarios)
+- [Acquiring Tokens](https://aka.ms/msal-net-acquiring-tokens)
 - [Username/password](https://aka.ms/msal-net-up)
-- [Customizing Token cache serialization](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/token-cache-serialization) (was not done in this sample, but you might want to add a serialized cache)
 - [Azure Active Directory v2.0 and the OAuth 2.0 resource owner password credential](https://review.docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) to learn more about the underlying protocol
 - [Resource owner password credentials RFC](https://tools.ietf.org/html/rfc6749#section-1.3.3)
 
-For more information about the Azure AD v2.0 endpoint see:
+For more information about the Microsoft identity platform (AAD) v2.0 see:
```
Comment (kalyankrishna1) [inline README.md:208]: >v2.0 [](start = 65, length = 4)

the v2.0 needs not be appended when using "Microsoft identity platform (AAD)" . #WontFix

**Review by kalyankrishna1 — APPROVED** [review]
:shipit:

File: README.md (hunk 2 above)
Comment (jmprieur) [reply README.md:208]: I'm surprised @kalyankrishna1 : AG  told me that …. #Resolved

Comment (kalyankrishna1) [reply README.md:208]: Please follow AG's advice Jean-Marc. I might have misunderstood

---
In reply to: [257620524](https://github.com/Azure-Samples/active-directory-dotnetcore-console-up-v2/pull/8#discussion_r257620524) [](ancestors = 257620524)

**Pull request merged**